api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -55,6 +55,10 @@ public Proxy(String id) {
 
     private String configServerPassword;
 
+    private Long lastCallTimeToAdmin;
```

Comment:
Is this variable necessary at all, since you already have minutesPastSinceLastContact?

I if yes, then I think the name should be more specific. Which time unit does it represent? Or is it a Date?

Reply:
So the lastCallTimeToAdmin is the last time the gateway called in a admin, here is saved as long, because it is a getTime() of Timestamp(In Milliseconds).
That is used for 2 parts:
-  The date on the proxies table/list (which is shown as a date -> this part is done by thymeleaf)
- To compute the minutesSinceLastContact.

Regarding the name, I agree with you, so I will proceed to change the variable name to a more specific name

Reply:
Imho, 'long' is a cumbersome way to express this variables purpose. It is a time/date and should therefore have a corresponding type. I suggest to use the LocalTime API which was also recommended in a comment below.

Reply:
Done